### ISSUE TITLE
Question About the loss during the Training

### ISSUE THREAD

**Issue by infiniteYuanyl** [hp-l33/ARPG#6]
When trainging a ARPG-XL model ,what is the training loss when it was nearly convergent?Below 1.0?Or a smaller number?I'm doing a training work on another dataset and it takes a long time on a single gpu.I'd like to know when it is about to be ok for inference.And thank you for your great works!!!

**Comment by hp-l33** [comment]
Thank you for your interest in our work!

Below is the final loss of our model on the ImageNet dataset (utilizing the llamagen tokenizer). 

| Model | Loss |
| ---- | ---- |
| ARPG-L | 7.50 |
| ARPG-XL | 7.30 |
| ARPG-XXL | 7.03 |

Based on my personal experience, using different datasets and different tokenizers may lead to significant differences in the numerical ranges of the loss values. Under our setup, the ARPG-L model is capable of generating reasonable images when it has been trained for 150 epochs, during which the loss is approximately 7.66.

**Comment by infiniteYuanyl** [comment]
I got it.Thank you very much!
